Source/Lib/pkg-config.pc.in

```diff
@@ -1,9 +1,9 @@
 prefix=@CMAKE_INSTALL_PREFIX@
-includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
+includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@/svt-hevc
 libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
 
 Name: SvtHevcEnc
 Description: SVT (Scalable Video Technology) for HEVC encoder library
 Version: 1.3.0
-Libs: -L${libdir} -lSvtHevcEnc -lpthread
+Libs: -L${libdir} -lSvtHevcEnc @LIBS_PRIVATE@
```

Comment:
If they're private libs only required for static linking the convention is to use `Libs.private`. This is supported by both pkg-config and pkgconf.

Reply:
Yea, I think I tried that before, but something failed. I can try again

Reply:
The ffmpeg build failed due to pthread stuff

Reply:
GStreamer plugin build pass with this PR.

Reply:
I noticed that the lib is not linked to pthread but the app is, is that ok?

Reply:
Isn't it always the executable which links all the dependencies? svt lib should not link to other libs.

Reply:
I see pthread stuff in some files in the codec folder

Reply:
I guess you are probably right

Reply:
I guess you are probably right

Reply:
Yeah, compiling this lib (the object code) only requires the headers, both for static or shared compilation; but static executables will need all the dependencies from this lib when linking.
So if svt requires Threads for static compilation, they should be specified on the pkg-config file, in a `Libs.private:` line.
When compiling the static executables that depend on svt, you then use `--static` flag, i.e. `pkg-config --static --libs`, to get both the libs in `Libs:` and the ones in `Libs.private:`.

Reply:
Although if I do link the lib to pthread and change the PKG config to libs. Private, ffmpeg compiled fine